CHANGELOG.md

```diff
@@ -20,7 +20,16 @@ The headers used in [Keep a Changelog](http://keepachangelog.com) are:
 - Fixed - for any bug fixes.
 - Security - in case of vulnerabilities.
 
-# [Unreleased](https://github.com/puppetlabs/beaker/compare/4.28.0...master)
+# [Unreleased](https://github.com/puppetlabs/beaker/compare/4.28.1...master)
```

Comment:
Please don't add an `Unreleased` entry when you're going to tag 4.28.1. Don't copy Puppetlabs' bad practices.

Reply:
🤷 I'm OK removing it.